examples/go-client/README.md

```diff
@@ -20,17 +17,70 @@ Auth API clients must perform two-way authentication using x509 certificates:
 2. They must offer their x509 certificate, which has been previously issued
    by the auth sever.
 
-Start up a teleport auth server and then run the following commands to create an `api-user` with a signed certificate. Make sure to run this from the `go-client` directory for proper output.
+There are a few simple ways to generate the certificates needed to authenticate the client, each of which is detailed in this demo.
```

Comment:
```suggestion
There are a few ways to generate the certificates needed to authenticate the client, each of which is detailed in this demo.
```
otherwise it implies that there are also hard methods which are not detailed. Correct me if that's intended as is though.